application/core/LSSodiumOld.php

```
@@ -115,7 +115,7 @@ public function encrypt($sDataToEncrypt)
      * Decrypt encrypted string.
      * @param string $sEncryptedString Encrypted string to decrypt
      * @param bool $bReturnFalseIfError false by default. If TRUE, return false in case of error (bad decryption). Else, return given $encryptedInput value
-     * @return string Return decrypted value (string or unsezialized object) if suceeded. Return FALSE if an error occurs (bad password/salt given) or inpyt encryptedString
+     * @return string Return decrypted value (string or unserialized object) if succeeded. Return FALSE if an error occurs (bad password/salt given) or input encryptedString
      * @throws SodiumException
      */
     public function decrypt($sEncryptedString, $bReturnFalseIfError = false): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$bReturnFalseIfError'. (undefined)

(UnusedFormalParameter)